### PR TITLE
Fix requestAnimationFrame timestamps in queue

### LINKED ISSUE #7044: Animation callbacks processed together should receive the same timestamp
In `Document::run_the_animation_frame_callbacks` we call `Performance::Now()` each time we invoke a callback. This contradicts [the specification](https://html.spec.whatwg.org/multipage/webappapis.html#run-the-animation-frame-callbacks). We can even write a test for this by adding two callbacks with `requestAnimationFrame` and asserting that the timestamp is equal. Use http://mxr.mozilla.org/servo/source/tests/wpt/web-platform-tests/animation-timing/ as a model.

Code: components/script/dom/document.rs

### PR DESCRIPTION
**Pull request by HarryLovesCode**
This resolves #7044 which involved callbacks in a queue not receiving the same timestamp despite the specification saying they should. An extra test was added to verify the correct behavior.

[Review on Reviewable](https://reviewable.io/reviews/servo/servo/7073)

### CODE REVIEW

File: tests/wpt/metadata/animation-timing/same-dispatch-time.html.ini [hunk 1]
```diff
@@ -0,0 +1,4 @@
+[same-dispatch-time.html]
+  type: testharness
+  [requestAnimationFrame will timestamp events in the same queue with the same time]
+    expected: PASS
```
Comment (SimonSapin) [inline tests/wpt/metadata/animation-timing/same-dispatch-time.html.ini]: I think that `PASS` is the default expectation, and this file can be removed entirely.

Comment (HarryLovesCode) [inline tests/wpt/metadata/animation-timing/same-dispatch-time.html.ini]: Yep, you are correct. I'll fix it in a second. Sorry about that!

**Comment by HarryLovesCode** [comment]
@SimonSapin ping

**Comment by HarryLovesCode** [comment]
@SimonSapin Does this need to be squashed?

**Comment by SimonSapin** [comment]
Feel free to squash, I don’t mind either way.

**Pull request merged**